### PR TITLE
Fix the world

### PR DESCRIPTION
**Pull request by acmcarther**
This makes a fix to serde to stop breaking things all the time.

@andrewmohrland
@seanstrom
@wmedrano

### CODE REVIEW

File: gemset.nix [hunk 1]
```diff
@@ -0,0 +1,66 @@
+{
```
Comment (seanstrom) [inline gemset.nix:1]: what's this?

File: default.nix [hunk 2]
```diff
@@ -0,0 +1,13 @@
+with (import <nixpkgs> {});
```
Comment (seanstrom) [inline default.nix:1]: what's this?

**Comment by acmcarther** [comment]
Also adds nix files for some reason

**Comment by ghost** [comment]
Abysmal. Three stars down. :(

**Pull request merged**